src/utils/utils.rs

```diff
@@ -538,10 +539,8 @@ pub fn format_path_for_display(path: &str) -> String {
 
 /// Encodes a utf-8 string as a null-terminated UCS-2 string in bytes
 #[cfg(windows)]
-pub fn string_to_winreg_bytes(s: &str) -> Vec<u8> {
-    use std::ffi::OsStr;
-    use std::os::windows::ffi::OsStrExt;
-    let v: Vec<u16> = OsStr::new(s).encode_wide().chain(Some(0)).collect();
+pub fn string_to_winreg_bytes(mut v: Vec<u16>) -> Vec<u8> {
```

Comment:
uhm, nope, lets not do this. I think we can just delete this helper if we switch to OsString.

Reply:
so AFAICT this is only used from windows.rs now, lets just include it in that file and remove it from utils.rs - it will be cleaner and less of an attractive nuisance I think.